docs/knowledge_base/tutorials/version_management.md

```diff
@@ -0,0 +1,104 @@
+When developing and deploying a Taipy application, it is straightforward to manage Taipy entities
+(scenarios, tasks, data nodes, etc.) and keeps them up-to-date when the configuration changes.
+
+In the following, we will use a basic Taipy application example defined in `main.py`.
+
+```python linenums="1" title="main.py"
+{%
+include-markdown "../../manuals/core/versioning/code_example/main.py"
+comments=false
+%}
+```
+
+Prerequisites of git is required to follow this tutorial.
+
+# Initialize a Taipy application
```

Comment:
Actually, you are initializing a git repository. 
What abt changing the tile to something like one of the following proposal: 
- Initialize a Git Repository
- Set up a Taipy application in Git

Reply:
How about "Set up the Taipy application as a Git repository"?

Because we can init git whenever, so it's kinda hard to give a general example

Reply:
Yes. I like it.

Reply:
```suggestion
# Set up the Taipy application as a Git repository
```